design-editor/src/pane/design-editor-element.js

```diff
@@ -30,10 +31,12 @@ import path, {relative, join} from 'path';
 
 const KEY_CODE = {
 	DELETE: 46
-};
-const INTERNAL_ID_ATTRIBUTE = 'data-id';
-const LOCK_CLASS = 'lock';
-const RE_COPY_ATTRIBUTE_TO_CONTAINER = new RegExp('^(data-(?!tau)|st-).*');
+	},
+	INTERNAL_ID_ATTRIBUTE = 'data-id',
+	LOCK_CLASS = 'lock',
+	RE_COPY_ATTRIBUTE_TO_CONTAINER = new RegExp('^(data-(?!tau)|st-).*'),
+	brackets = window.brackets || window.top.brackets,
```

Comment:
Please use our function `checkGlobalContext()` from utils module to choose if some package get from window or window top.
Example
``` javascript
brackets = utils.checkGlobalContext('brackets');
```